db/index.js

```diff
@@ -3,13 +3,15 @@ const mongoose = require('mongoose');
 const db = mongoose.connection;
 
 // Keeping in comments for future use.
-// var sizingSchema = new mongoose.Schema({
-//   size: { type: String },
-//   neck: String,
-//   chest: String,
-//   sleeve: String,
-//   type: { type: String, default: 'Men - Shirts' }
-// });
+const sizingSchema = new mongoose.Schema({
+  size: { type: String },
```

Comment:
Why is size type defined in an object where neck, chest, and sleeve isn't?

Reply:
Changed that.